ci/README.md

```diff
@@ -0,0 +1,15 @@
+## Overview
+Travis integration is prepared to check pull request with different tests. It uses docker images as clean environment with all build dependencies onboard. 
```

Comment:
I propose the following rewording:

> Travis is enabled in this repository so each pull request is checked before being allowed to merge. The system is based on fiware/orion-ci:rpm7 which is automatically built each nigth from fiware-orion repository master branch, providing a clean environment with all build dependencies onboard. The Dockerfile used to build this docker is available in the ci/rpm7 directory.
> 
> The Travis checks are divided into stages, which are described in "Supported tests" section

@caa06d9c is correct what I say above about nigthly frecuency and master branch usage?

Reply:
No. This image is stable and rebuilds on demand. It can be an automatic process, not a problem. Main rpm package is built every night from the master branch and the same approach should be done for the main one Orion docker image, but it's a new question and should be discussed before.

Reply:
fiware/orion (the main Orion docker) image is rebuild each time a new PR lands in master. Maybe the same appraoch should be done for fiware/orion-ci:rpm7. In that case, use that in the text, I mean:

"... which is build from master branch each time a new PR lands in master"